app/models/manageiq/providers/vmware/infra_manager/provision/cloning.rb

```diff
@@ -47,10 +47,6 @@ def prepare_for_clone_task
       cluster ? cluster.default_resource_pool : dest_host.default_resource_pool
     end
```

Comment:
@bzwei In a separate PR, can we create a similar `dest_resource_pool` method and move all this logic there?